templates/miq-template.yaml

```diff
@@ -691,10 +771,14 @@ parameters:
   displayName: Memcached Slab Page Size
   description: Memcached size of each slab page.
   value: 1m
+- name: POSTGRESQL_CONFIG_DIR
+  displayName: PostgreSQL Configuration Overrides
+  description: Directory used to store PostgreSQL configuration overrides.
+  value: "/etc/manageiq/postgresql.conf.d"
```

Comment:
If this directory doesn't exist, will the config map mount create the parent directory?